distribution/pom.xml

```diff
@@ -97,8 +97,41 @@
     <dependency>
       <groupId>org.apache.hbase</groupId>
       <artifactId>hbase-client</artifactId>
+      <exclusions>
+       <exclusion>
+           <groupId>io.netty</groupId>
+           <artifactId>netty</artifactId>
+       </exclusion>
+      </exclusions>
+    </dependency>
+    <dependency>
```

Comment:
Not sure why we need to netty dependency with `test` scope in distribution pom...